courses/free-courses-pt_BR.md

```diff
@@ -181,6 +182,7 @@
 * [Curso Intensivo de Next.js & React](https://www.cod3r.com.br/courses/curso-intensivo-next-react) - Leonardo Leitão (Cod3r)
 * [Curso Starter ES6, Node, React e React Native](https://app.rocketseat.com.br/discover/courses) - Rocketseat
 * [Fast & Furious](https://www.youtube.com/playlist?list=PLy5T05I_eQYOoUz2TtAqq35RLCc-xBZCe) - Codecasts
+* [Discover](https://app.rocketseat.com.br/discover) - Rocketseat
```

Comment:
Arrangement in alphabetical order is required.

Reply:
not a direct link - can't use